layouts/community/ergodox/coderkun_neo3/keymap.c

```diff
@@ -0,0 +1,288 @@
+#include QMK_KEYBOARD_H
```

Comment:
This file needs a GPLv2-compatible license header.

* [PR Checklist - Example GPLv2 Header - QMK Firmware Documentation](https://docs.qmk.fm/#/pr_checklist?id=example-gplv2-header)